.circleci/config.yml

```diff
@@ -121,7 +121,6 @@ jobs:
           path: test-results
 
 workflows:
-  version: 2
```

Comment:
For future reference, `version: 2` can be removed because the overall config version is `2.1`https://circleci.com/docs/configuration-reference#workflow-version).